src/Symfony/Bundle/FrameworkBundle/CacheWarmer/RouterCacheWarmer.php

```diff
@@ -26,12 +26,11 @@
  */
 class RouterCacheWarmer implements CacheWarmerInterface, ServiceSubscriberInterface
 {
-    private ContainerInterface $container;
-
-    public function __construct(ContainerInterface $container)
+    /**
+     * As this cache warmer is optional, dependencies should be lazy-loaded, that's why a container should be injected.
+     */
+    public function __construct(private ContainerInterface $container)
```

Comment:
I think we should use multi line signatures all the time when using CPP (with trailing comma)